google/cloud/pubsub/internal/subscription_lease_management_test.cc

```diff
@@ -65,7 +65,7 @@ TEST(SubscriptionLeaseManagementTest, NormalLifecycle) {
           EXPECT_LE(std::abs((kTestDeadline - extension).count()), 2);
           return make_ready_future(Status{});
         });
-    // Then a message is nacked.
+    // Then a message is Nacked.
```

Comment:
Capitalizing `Nacked` looks funny. It's a verb. Perhaps we should leave these lowercase and add them to the ignore list?

Reply:
OK I implemented those changes. I'll wait for the other PR to get merged and rebase before uploading the new changes.